meta-client/src/main/java/org/apache/helix/metaclient/factories/MetaClientConfig.java

```diff
@@ -145,6 +174,18 @@ final B self() {
     }
 
     protected void validate() {
+      if (_metaClientReconnectPolicy == null) {
+        _metaClientReconnectPolicy = new ExponentialBackoffReconnectPolicy();
```

Comment:
Neat : looks like we have default retry policy as exponential polity. May be we can comment in this Metaclientretry policy